onnx/defs/parser.h

```diff
@@ -160,6 +160,15 @@ class KeyWordMap {
     return KeyWord::NONE;
   }
 
+  static const std::string& ToString(KeyWord kw) {
+    static std::string undefined("undefined");
+    for (const auto& pair : Instance()) {
```

Comment:
Maybe it would be worth to have a dictionary value: key to speed it up? I don't know how many times this is called.

Reply:
May be. Doesn't seem critical to me yet. We can do this later, if needed.